src/plugins/intel_cpu/src/transformations/snippets/aarch64/pass/snippets_mark_skipped.cpp

```diff
@@ -93,9 +95,9 @@ bool isSuitableBinaryConvolutionParent(const std::shared_ptr<const Node>& node)
     return is_suitable_node && has_only_child;
 }
 bool isSuitableMiscParent(const std::shared_ptr<const Node>& node) {
-    const bool is_suitable_node = ov::is_type<ov::op::v0::NormalizeL2>(node) ||
-                                  ov::is_type<ov::op::v1::ConvolutionBackpropData>(node) ||
-                                  ov::is_type<ov::op::v1::GroupConvolutionBackpropData>(node);
+    const bool is_suitable_node = ov::is_type_any_of<ov::op::v0::NormalizeL2,
+                                                     ov::op::v1::ConvolutionBackpropData,
+                                                     ov::op::v1::GroupConvolutionBackpropData>(node);
```

Comment:
Backprop convolution have common base, Maybe is enough to check it?

Reply:
Yes, checks for `ov::op::v1::ConvolutionBackpropData` and `ov::op::v1::GroupConvolutionBackpropData` are merged